ports/stm32/main.c

```diff
@@ -644,7 +644,7 @@ void stm32_main(uint32_t reset_mode) {
     // if an SD card is present then mount it on /sd/
     if (sdcard_is_present()) {
         // if there is a file in the flash called "SKIPSD", then we don't mount the SD card
-        if (!mounted_flash || mp_vfs_import_stat("SKIPSD") == MP_IMPORT_STAT_FILE) {
+        if (!mounted_flash || mp_vfs_import_stat("SKIPSD") == MP_IMPORT_STAT_NO_EXIST) {
```

Comment:
That would allow a directory called SKIPSD to skip mounting as well (compared to `... != MP_IMPORT_STAT_FILE`).  I guess that's ok, more general.